EVENT.md

```diff
@@ -1110,7 +1110,10 @@ sites represented by unexpired sources for a source-site.
 The browser can place a limit on the number of a source site's unexpired source's
 unique `destination` sites. When an attribution source is registered for a site
 that is not already in the unexpired sources and a source site is at its limit,
-the browser will drop the new source.
+the browser will only count the most recent `destination` sites within this
+limit, including those registered with the new source. The browser will delete
+pending sources and reports associated with the `destination` sites that are no
```

Comment:
If reports were already delivered for the source/s with pending reports, does this still address the history reconstruction attacks?

Why is no consideration given to the volume of sources and reports that may be deleted with strict FIFO?

Reply:
@vikassahu29 may have better explanation, but with FIFO we ensure that at any point there are no more than 100 destinations sent in the reports for unexpired sources.

We would like to keep this limit purely based on time to be privacy neutral. Utility-wise, we expect that ad-techs most of time would favor more recent sources.

Reply:
Utility-wise, I would also expect API consumers to favour receiving reports. More recent sources don't necessarily guarantee that, and the volume of potential report deletion seems unaccounted for.

Reply:
The volume of potential report deletion definitely contains cross-site leak, and I don't think we would consider this as a rule for FIFO.

Reply:
Sources would be used for future attributions/reports though, and that's what we would like to limit.

Reply:
My understanding is we want to "ensure that at any point there are no more than 100 destinations sent in the reports." Regardless of what sources and triggers the user-agent gets, can't this be controlled by the reports themselves, theoretically?